src/hooks/useFeaturesSort.ts

```diff
@@ -15,6 +15,7 @@ export type FeaturesSortType =
     | 'enabled'
     | 'stale'
     | 'created'
+    | 'archived'
```

Comment:
I think entire `useFeaturesSort` unnecessary. `ArchiveListContainer` is not in use anywhere.

Reply:
To add to this: If there are components not in use as a result of this PR replacing them, it would be good to delete them in the same PR.